kolibri/plugins/management/assets/src/device_management/views/manage-content-page/wizards/wizard-import-network.vue

```diff
@@ -9,32 +9,34 @@
     @enter="submit"
     @back="goBack"
   >
-    <div class="main">
-      <k-textbox :label="$tr('enterContentChannel')" v-model="channelId" :disabled="wizardState.busy"/>
-    </div>
-    <div class="core-text-alert">
-      {{ wizardState.error }}
-    </div>
-    <div class="button-wrapper">
-      <k-button
-        @click="cancel"
-        :text="$tr('cancel')"
-        :raised="false"
-        :disabled="wizardState.busy"/>
-      <k-button
-        :text="$tr('import')"
-        @click="submit"
-        :disabled="!canSubmit"
-        :primary="true"/>
-    </div>
+    <form @submit.prevent="submit">
+      <div class="main">
+        <k-textbox :label="$tr('enterContentChannel')" v-model="channelId" :disabled="wizardState.busy"/>
```

Comment:
Needs validation

Reply:
Reference the [class-create-modal](https://github.com/learningequality/kolibri/blob/release-v0.6.x/kolibri/plugins/management/assets/src/views/manage-class-page/class-create-modal.vue) for validation flow.